### PR TITLE
Expand documentation of CCS

### LINKED ISSUE #97: CCS Readme
Create a Readme that has a high level overview of the functionality of the CCS.

### PR DESCRIPTION
**Pull request by bill-luu**
Gave information on the horn activity, as well as specified which CAN network to use.

Fixes #97

### CODE REVIEW

File: Documentation/CCS.adoc [hunk 1]
```diff
@@ -1,10 +1,21 @@
-CCS Software
--------------
+# CCS Software
 
-The CCS board will be connected to the CAN bus as well as output the telemetry stream to the Primary Pi. 
+The primary responsibility of the CCS is to take all the information on the 500Khz CAN Network (CAN2), package the information into serial data, and send it to the Raspberry Pi over UART.
 
-The responsibility of CCS is to pass information from the CAN network into the telemetry stream to be forwarded out. 
+In addition, the CCS also recieves messages from the driver controls, and will activate the horn when it is pressed.
```
Comment (safasyed) [inline Documentation/CCS.adoc]: receives*

File: Documentation/CCS.adoc [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 
+## Serial Output
 
+See the https://docs.google.com/spreadsheets/d/1gtCKN7LzG7e8XRzbGrN5uUiLGr7miS0QCEm0QNylUyM/edit#gid=0[Elysia Communications Protocol] for messages that the CCS forwards to the Raspberry Pi & the breakdown of the serial packets.
+
+## Unused CAN Network
+
+CAN1 was originally intended to house older external systems(eg. Tritium BMS) that could only run on 250Khz networks.
+However, those have been replaced with systems that can be run on 500Khz.
+As a result, CAN1 is currently being used by the Software Team.
```
Comment (safasyed) [inline Documentation/CCS.adoc]: Specify what CAN1 is being used for by the Software team

Comment (adolapo) [reply Documentation/CCS.adoc]: Yeah that would be helpful

Comment (bill-luu) [reply Documentation/CCS.adoc]: Oops, I meant *not* used by the software team :laughing:

**Pull request merged**